html/screenreaders/demos/list-with-links/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Список со ссылками — Скринридеры — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    .body {
```

Comment:
```suggestion
    body {
```

Нет особого смысла обращаться к уникальному объекту по классу =)